src/Api/Model/Languageforge/Lexicon/Command/LexUploadCommands.php

```diff
@@ -125,11 +126,16 @@ public static function uploadAudioFile($projectId, $mediaType, $tmpFilePath)
 
             }
 
+            //Find the size of the upload file now pointed to by the entry
+            $fileSize = filesize($filePath);
+
+
             // construct server response
             if ($moveOk && $tmpFilePath) {
                 $data = new MediaResult();
                 $data->path = $project->getAudioFolderPath($project->getAssetsRelativePath());
                 $data->fileName = $fileNamePrefix . '_' . $fileName; //if the file has been converted, $fileName = converted file
+                $data->fileSize = $fileSize;
```

Comment:
Now that calculating the filesize is trivial, let's do it all on one line
```suggestion
                $data->fileSize = filesize($filePath);
```